lib/datasource/remote/api/eos_repository.dart

```diff
@@ -28,12 +28,22 @@ abstract class EosRepository {
   String actionNameAcceptnew = 'acceptnew';
   String actionNameRecover = 'recover';
   String actionNameUnplant = 'unplant';
+  String proposalActionNameDelegate = 'delegate';
+  String proposalActionNameUndelegate = 'undelegate';
 
   // Authorizations
   String permissionActive = 'active';
   String permissionOwner = 'owner';
   String permissionApplication = 'application';
 
+  // Voice scopes
+  List<String> voiceScopes = [
```

Comment:
should this list be private? is it only used here?

Reply:
Its used twice.
![image](https://user-images.githubusercontent.com/42857405/135377202-50a733ca-7fca-4f69-9978-6a7795043841.png)

Reply:
for proposal repo, since its only used there

Reply:
but im also cool with it here. No blocker.